lib/reporter.js

```diff
@@ -152,15 +152,25 @@ module.exports = class Reporter {
     if (user.enabled === false) {
       return;
     }
-
+    const login = user.login.toLowerCase();
+    if (this.jobs[login] === undefined) {
+      this.jobs[login] = [];
+    }
     const date = moment(time, 'HH:mm')
       .utcOffset(user.timezone, true)
       .utcOffset(moment().utcOffset());
 
-    const rule = new schedule.RecurrenceRule(null, null, null, null, date.hour(), date.minute());
+    const rule = new schedule.RecurrenceRule(
+      null,
+      null,
+      null,
+      [new schedule.Range(1, 5)], // 1 = Mon, 5 = Fri
```

Comment:
This will break due to timezone issues. To demonstrate, assume our server is running in San Francisco and we're sending reports at 08:00am. 08:00am CET (e.g. Vienna) is 23:00pm on the *previous day* in PST, so we're skipping the Monday report.